ERCS/erc-7066.md

```diff
@@ -47,15 +47,15 @@ This proposal is different from other locking proposals in number of ways:
 - This implementation proposes a separation of rights between locker and approver. Token can be locked with approval and approved can unlock and withdraw tokens (opening up opportunities like renting, lending, bnpl etc), and token can be locked lacking the rights to revoke token, yet can unlock if required (opening up opportunities like account-bound NFTs).
```

Comment:
```suggestion
- This implementation proposes a separation of rights between locker and approver. Token can be locked with approval and approved can unlock and withdraw tokens (opening up opportunities like renting, lending, BNPL etc), and token can be locked lacking the rights to revoke token, yet can unlock if required (opening up opportunities like account-bound NFTs).
```

Also, please include the abbreviation on line 38:

> - Buy Now Pay Later (BNPL): ...

We prefer that abbreviations are expanded the first time they are used.